File: tests/fixtures/sample.java
// A sample Java file for testing TODO detection

public class Sample {
    // TODO: Add logging framework
    public static void main(String[] args) {
        System.out.println("Hello");
    }

    // BUG(dave, #789): NullPointerException when input is empty
    public void process(String input) {
        input.length();
    }
}

File: tests/fixtures/sample.py
# A sample Python file for testing TODO detection

def main():
    # TODO: Add type hints
    x = 42

    # FIXME(alice, p:high): This needs better error handling
    try:
        y = x / 0
    except:
        pass

    # BUG: Division by zero not properly handled
    print(x)

if __name__ == "__main__":
    main()
